.licensei.toml

```diff
@@ -9,4 +9,5 @@ approved = [
 ignored = [
   "github.com/gogo/protobuf",
   "sigs.k8s.io/yaml",
+  "github.com/oklog/run",
```

Comment:
Is this really necessary? Apache 2.0 license is approved.

Reply:
```
github.com/oklog/run: no license file found
make: *** [license-check] Error 2
```